packages/protocol/scripts/foundry/run_integration_tests_in_anvil.sh

```diff
@@ -8,6 +8,11 @@ source $PWD/scripts/foundry/constants.sh
 echo "Generating and running devchain before running integration tests..."
 source $PWD/scripts/foundry/create_and_migrate_anvil_devchain.sh
 
+# Before running the migration script, check that the tests compile
+# This will avoid having to wait for the whole chain to start just to realize
+# there's a compilation error at the very end
+forge build --contracts $PWD/test-sol/integration
```

Comment:
Nit: Explicitly building the contracts here might not be necessary in this script since it runs `create_and_migrate_anvil_devchain.sh` which has the same call:

https://github.com/celo-org/celo-monorepo/blob/965bb5734a527ee2801fd08eacd4032963e22977/packages/protocol/scripts/foundry/create_and_migrate_anvil_devchain.sh#L31

```suggestion
source $PWD/scripts/foundry/create_and_migrate_anvil_devchain.sh
```